### PR TITLE
{AKS} Remove aks_create_with_loadbalancer_and_update_to_none_outbound

### PR DESCRIPTION
**Pull request by MartinForReal**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

remove `aks_create_with_loadbalancer_and_update_to_none_outbound` because feature is not ready

### Related command

az aks

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Review by FumingZhang — COMMENTED** [review]
Please also remove the corresponding recording file `src/aks-preview/azext_aks_preview/tests/latest/recordings/test_aks_create_with_loadbalancer_and_update_to_none_outbound.yaml`.

**Comment by FumingZhang** [comment]
And clarify the reason to remove the test case.

**Review by FumingZhang — APPROVED** [review]
LGTM

File: src/aks-preview/HISTORY.rst [hunk 1]
```diff
@@ -8,6 +8,9 @@ Guidance
 If there is no rush to release a new version, please just add a description of the modification under the *Pending* section.
 
 To release a new version, please select a new version number (usually plus 1 to last patch version, X.Y.Z -> Major.Minor.Patch, more details in `\doc <https://semver.org/>`_), and then add a new section named as the new version number in this file, the content should include the new modifications and everything from the *Pending* section. Finally, update the `VERSION` variable in `setup.py` with this new version number.
+Pending
++++++++
+remove aks_create_with_loadbalancer_and_update_to_none_outbound because feature is not ready
```
Comment (yanzhudd) [inline src/aks-preview/HISTORY.rst:13]: I'm not sure if we need to add this change to the history or not since it seems not a customer-facing change.

Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:13]:
```suggestion
```

**Review by yanzhudd — APPROVED** [review]

**Comment by yanzhudd** [comment]
please fix the CI issues
image

**Pull request merged**